build/wrapper.cc

```diff
@@ -110,13 +110,18 @@ void initialize_worker(
                         application_error,
                         is_retryable_error);
 
-          RAY_CHECK(return_vec.size() == 1);
-
           // TODO: support multiple return values
           // https://github.com/beacon-biosignals/Ray.jl/issues/54
-          (*returns)[0].second = return_vec[0];
-          return Status::OK();
+          if (return_vec.size() == 1) {
+            (*returns)[0].second = return_vec[0];
+            return Status::OK();
+          } 
+          else {
+            auto msg = "Task returned " + std::to_string(return_vec.size()) + " values. Expected 1.";
+            return Status::NotImplemented(msg);
+          };
```

Comment:
I don't think there's any need for this restriction anymore... I'll make a quick attempt to obviate it

Reply:
moving this into https://github.com/beacon-biosignals/Ray.jl/pull/193 as it contains more changes than I think is suitable for this PR